apps/omg_api/lib/state/transaction.ex

```diff
@@ -26,11 +26,12 @@ defmodule OMG.API.State.Transaction do
   @max_inputs 4
   @max_outputs 4
 
-  defstruct [:inputs, :outputs]
+  defstruct [:inputs, :outputs, metadata: ""]
 
   @type t() :: %__MODULE__{
           inputs: list(input()),
-          outputs: list(output())
+          outputs: list(output()),
+          metadata: bitstring()
```

Comment:
Difference between `bitstring` and `binary` is as follows:
```
binary()    <<_::_*8>>
bitstring()    <<_::_*1>>
```
We should use the narrowest type possible. In this case - `binary()`.